pages/index.js

```diff
@@ -281,8 +283,11 @@ function FaqList({ title, list }) {
   );
 }
 
+// const what = withRouter.query;
 // Main ------------------------------------------------------------------------
 export default function Home() {
+
+  const REFERRAL = typeof window !== 'undefined' ? `?prefill_Source=${encodeURIComponent(location.search)}` : "?prefill_Source=Louisiana%20Health%20Work%20Connect%20Website";
```

Comment:
You want to extract the `source` parameter from the query string and use that as the `prefill_Source` parameter in the URL you're manufacturing. 

Right now, if say `https://healthworkconnect.la.gov/?this=is&a=test` is the URL, then `window.location.search` is `"?this=is&a=test"` and that is the value that will wind up prefilled in source. That won't be super useful on the backend.